### PR TITLE
#5470 : Incorrect NLS msg; should be create action vs --create option

### LINKED ISSUE #5470: NLS message CWWKE0031E is inaccurate when emitted from server script
The error message 
error.noExistingServer=CWWKE0031E: Specified server {0} does not exist; use the --create option to create a new server. serverPath: {1}
is incorrect, at least in the context of the server/server.bat command line script where issuing a command like:
**./server dump**
(with no server name and where the defaultServer doesn't exist)
results in 
**CWWKE0005E: The runtime environment could not be launched.
CWWKE0031E: Specified server defaultServer does not exist; use the --create option to create a new server. serverPath: ...**
but the server script has no **--create** option, only **create**.  Oddly enough, specifying a non-existent server name like:
**./server dump foo**
results only in output from server script:
**Server foo does not exist.**

### PR DESCRIPTION
**Pull request by hlhoots**
#build 
fixes #5470

### CODE REVIEW

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -151,7 +151,7 @@ error.creatingNewServer=CWWKE0030E: An exception occurred while creating server
 error.creatingNewServer.explanation=The specified server could not be created.
 error.creatingNewServer.useraction=Check and resolve the exception, then try again to create the server.
 
-error.noExistingServer=CWWKE0031E: Specified server {0} does not exist; use the --create option to create a new server. serverPath: {1}
+error.noExistingServer=CWWKE0031E: Specified server {0} does not exist; use the create action to create a new server. serverPath: {1}
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops:154]: Specified server {0} does not exist
-->
Specified {0} server  does not exist

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 2]
```diff
@@ -488,7 +488,7 @@ error.creatingNewClient=CWWKE0902E: An exception occurred while creating client
 error.creatingNewClient.explanation=The specified client could not be created.
 error.creatingNewClient.useraction=Check and resolve the exception, then try again to create the client.
 
-error.noExistingClient=CWWKE0903E: Specified client {0} does not exist; use the --create option to create a new client. clientPath: {1}
+error.noExistingClient=CWWKE0903E: Specified client {0} does not exist; use the create action to create a new client. clientPath: {1}
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops:491]: Specified client {0} does not exist
--->
Specified {0} client does not exist

**Review by dmuelle — CHANGES_REQUESTED** [review]
your edit looks good, the existing message needs to change to be consistent with ID guidelines if it's possible to make that edit here

**Comment by hlhoots** [comment]
#build 
rebuild for requested changes above from David

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops (hunk 1 above)
Comment (hlhoots) [reply dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: fixed

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops (hunk 2 above)
Comment (hlhoots) [reply dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: fixed

**Comment by hlhoots** [comment]
@dmuelle made the updates you requested. thanks for the quick review!

**Review by dmuelle — APPROVED** [review]
LGTM

**Comment by hlhoots** [comment]
Unit test failure that seems unrelated.  All FATs ran okay.  Rebuilding.

#build

**Comment by hlhoots** [comment]
rebasing

#build

**Comment by skasund** [comment]
L2 message review completed

**Comment by hlhoots** [comment]
Last build had a couple of errors but look unrelated to this message change.

**Review by brenthdaniel — APPROVED** [review]

**Comment by hlhoots** [comment]
#run-libby-bot

**Pull request merged**